7024Movies/Pages/DiscoverComedy.cshtml

```diff
@@ -20,7 +20,7 @@
     <a class="center btn btn-dark" href="/MovieReviewsComedy/Index" role="button">Audience Reviews</a>
 </div>
 <div class="row padbottomextra">
-    <table class="table sortable table-sm table-hover" width="100%">
+    <table class="table sortable table-sm table-hover" width="100%" border="1";>
```

Comment:
Best to put this in a linked stylesheet, so 1) it is consistent across pages, and 2) you don't bloat your content with style.